### PR TITLE
Clarify installation + configuration

### PR DESCRIPTION
**Pull request by inverse**
Related to #167

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,9 +5,15 @@
...
+## Installation
 
-Hue dimmer remotes can be used for a click and long press (hold button for 2 sec and see LED blink twice).
+Place the `custom_components` folder in your configuration directory (or add its contents to an existing `custom_components` folder). You need to set up your [Hue bridge](https://www.home-assistant.io/integrations/hue) first.
+
+Alternatively it can be installed via [HACS](https://hacs.xyz/).
+
+## Configuration
+
+Once installed you need to setup the `huesensor` integration in your configuration.
```
Comment (inverse) [inline README.md]: @robmarkcole I think we should also explain why each entry is configured.

Comment (robmarkcole) [reply README.md]: what do you suggest we add?

Comment (inverse) [reply README.md]: Perhaps explaining why adding each one... e.g what it gives. Perhaps It's not required though :+1:

Comment (robmarkcole) [reply README.md]: can open that as a second PR if you think it has merit :-)

**Pull request merged**